articles/howtos/toc.yml

```
@@ -0,0 +1,4 @@
+- name: How do I make a Template, Decorator Ready?
```

```suggestion
- name: How do I make a Template able to support extendability through Decorators?
```

For maintainability purposes, is it possible for the `name`s to be blank and have DocFX automatically populate it with the article title from an `xref`? This may require that we change TOC files from `.yml` to `.md`.